tests/integration/targets/mso_schema_site_bd_l3out/tasks/main.yml

```diff
@@ -58,6 +58,7 @@
     - '{{ mso_site | default("ansible_test") }}'
     - 'aws_{{ mso_site | default("ansible_test") }}'
     - 'azure_{{ mso_site | default("ansible_test") }}'
+  ignore_errors: yes
```

Comment:
Technically, this case shouldn't fail right?

Reply:
have removed was due to azure site not being reachable and thus failing